WordPress/Classes/Models/WPAccount.h

```diff
@@ -11,24 +13,29 @@
 /// @name Properties
 ///-----------------
 
-@property (nonatomic, strong)   NSNumber    *userID;
-@property (nonatomic, strong)   NSString    *avatarURL;
-@property (nonatomic, copy)     NSString    *username;
-@property (nonatomic, copy)     NSString    *uuid;
-@property (nonatomic, strong)   NSDate      *dateCreated;
-@property (nonatomic, strong)   NSString    *email;
-@property (nonatomic, strong)   NSString    *displayName;
-@property (nonatomic, strong)   NSNumber    *emailVerified;
-@property (nonatomic, strong)   NSNumber    *primaryBlogID;
-@property (nonatomic, strong)   NSSet<Blog *>       *blogs;
-@property (nonatomic, strong)   Blog        *defaultBlog;
-@property (nonatomic, strong)   ManagedAccountSettings *settings;
+/*
+ The nullablity annotation on the Core Data properties follows the data scheme in the model file. Some of these
+ annotations do not make much sense (like how can userId be nullable?). But we'll follow the Core Data scheme to be
+ extra safe.
+ */
+
+@property (nonatomic, strong, nullable)   NSNumber    *userID;
+@property (nonatomic, strong, nullable)   NSString    *avatarURL;
+@property (nonatomic, copy, nonnull)     NSString    *username;
```

Comment:
(nit) `nonnull` is redundant.

Reply:
Are we sure `username` is `nonnull`?

Reply:
Yes, according to the Core Data schema.

<img width="1060" alt="Screenshot 2025-04-01 at 1 59 16 PM" src="https://github.com/user-attachments/assets/5e43b192-eb97-4cec-8663-1cc483763bb7" />